BUILD.bazel

```diff
@@ -46,11 +46,16 @@ go_binary(
     embed = [":go_default_library"],
     # These gc_linkopts make it a static binary. See
     # https://github.com/bazelbuild/rules_go/issues/161#issuecomment-304469169.
-    gc_linkopts = [
-        "-linkmode",
-        "external",
-        "-extldflags",
-        "-static",
-    ],
+    gc_linkopts = select({
```

Comment:
I think we can just remove `gc_linkopts` entirely. `cip` appears to be a pure go binary, so it'll be statically linked by default.

Reply:
we can also force it to be a pure go binary by setting `pure = "true"`, but I don't think we need or want that here.

Reply:
Ack